app/client/components/sectionHeader.tsx

```diff
@@ -38,47 +57,40 @@ export const SectionHeader = (props: SectionHeaderProps) => (
     >
       <div
         css={{
-          display: "none",
+          display: "block",
```

Comment:
Ello ... is setting this property necessary? div has a default display property of block.

Reply:
Nah this isn't necessary, a remnant of refactoring